framework/source/class/qx/test/ui/tree/virtual/WidgetProvider.js

```diff
@@ -286,7 +286,7 @@ qx.Class.define("qx.test.ui.tree.virtual.WidgetProvider",
       var newWidget2Bindungs = widget2.getBindings().length;
       var newModelBindungs = this.getLookupTable().getBindings().length;
       this.assertEquals(oldWidget1Bindungs, newWidget1Bindungs, "Binding on first widget is not removed!");
-      this.assertEquals(oldWidget1Bindungs, newWidget1Bindungs, "Binding on second widget is not removed!");
+      this.assertEquals(oldWidget2Bindungs, newWidget2Bindungs, "Binding on second widget is not removed!");
```

Comment:
Shouldn't we, while we're fixing these lines, fix the spelling of Bindings in this function???